probe-rs/src/bin/probe-rs/cmd/cargo_embed/mod.rs

```diff
@@ -227,12 +226,39 @@ fn main_try(mut args: Vec<OsString>, offset: UtcOffset) -> Result<()> {
     };
 
     if config.flashing.enabled {
-        flash(
-            &config,
+        // As opposed to cargo-flash, we do not have the option of suppliying an external image.
+        // This means we can just take the cargo target, if it's set.
+        let image_instr_set = opt
```

Comment:
Can we leave this in the separate function to not let our functions grow even bigger than they already are? :D

Reply:
Good call, also I need to make the new dep optional

Reply:
Oh, you meant _that_ part... Eeeeh IDK I kinda want to pull loader/flashing together with at least cargo-flash so I'd probably keep it like this for now.